core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -556,7 +556,7 @@ public void deleteUser(UUID id, Optional<String> message)
 		MessageSource anonymousMessageSource = messageSourceRepository.findOne(userEntity.getAnonymousMessageSourceId());
 		UserAnonymized userAnonymizedEntity = userAnonymizedService.getUserAnonymizedEntity(userAnonymizedId);
 		userAnonymizedEntity.clearAnonymousDestination();
-		userAnonymizedEntity = userAnonymizedService.updateUserAnonymized(userAnonymizedEntity);
+		userAnonymizedService.updateUserAnonymized(userAnonymizedEntity);
```

Comment:
The `userAnonymizedEntity` is later on in this method saved again. Should this method not return the updated entity to be used in the next save?

Reply:
It doesn't matter, as the entity itself isn't updated. If it would matter, we would need to do a ``getOne`` to reload it from Hibernate. We can't return the entity from the ``UserAnonymizedService``, as that returns cacheable DTOs only